components/ntp_background_images/common/pref_util.cc

```diff
@@ -0,0 +1,29 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/ntp_background_images/common/pref_util.h"
+
+#include "base/check.h"
+#include "brave/components/ntp_background_images/common/features.h"
+#include "brave/components/ntp_background_images/common/pref_names.h"
+#include "components/prefs/pref_service.h"
+
+namespace ntp_background_images::prefs {
```

Comment:
Maybe we don’t need the prefs namespace here. `ntp_background_images::prefs::ShouldShowNewTabTakeoverInfobar(prefs)` seems verbose. Something like `ntp_background_images::ShouldShowNewTabTakeoverInfobar(prefs)` might be clearer. Your thoughts?

Reply:
Maybe this would make sense as a `*_infobar_util` ?